handler/challenge.go

```diff
@@ -0,0 +1,12 @@
+package handler
+
+import (
+	"net/http"
+	"os"
+)
+
+// HandleChallenge will return the value in the environ variable ACME_CHALLENGE for let's encrypt
+func (h *Handler) HandleChallenge(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte(os.Getenv("ACME_CHALLENGE")))
```

Comment:
Should we add something to `app.env.dist` as well? I guess we can revisit that at some point.

Reply:
Made #50 for that. This is gtg.